api/v1alpha1/containerimagebuild_types.go

```diff
@@ -72,7 +74,19 @@ type Registry struct {
 	DynamicCloudCredentials bool `json:"dynamicCloudCredentials"`
 }
 
-// ContainerImageBuildSpec defines the desired state of ContainerImageBuild
+// ContainerImageBuildMessaging controls the parameters used to send build status updates over a message bus.
+type ContainerImageBuildMessaging struct {
+	// MessageBroker indicates the message bus implementation.
+	MessageBroker message.Broker `json:"broker"`
+
+	// AMQPURI designates the AMQP broker connection URI.
+	AMQPURI string `json:"amqpURI"`
```

Comment:
I don't know if I love that this would have the password built-in. I didn't love that it was in the controller deployment either, but at least I could pretend we could move it to a secret and configure it in the helm chart easily.

Reply:
Well, I can migrate either the credentials or the full string into a secret? Let me think about this for a bit.